src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -119,7 +119,8 @@
   --pf-c-menu-toggle--m-split-button--child--disabled--Color: var(--pf-global--Color--dark-200);
   --pf-c-menu-toggle--m-split-button--child--disabled--BackgroundColor: var(--pf-global--disabled-color--300);
   --pf-c-menu-toggle--m-split-button--first-child--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-menu-toggle--m-split-button--last-child--PaddingLeft: 0;
+  --pf-c-menu-toggle--m-split-button--last-child--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-menu-toggle--m-split-button--last-child--PaddingRight: var(--pf-global--spacer--md);
```

Comment:
This is now targeting the controls element. This makes the button inside tiny -

<img width="458" alt="Screen Shot 2022-03-28 at 5 29 17 PM" src="https://user-images.githubusercontent.com/35148959/160497786-397e7491-8b83-4405-9dea-b66171150de6.png">
<img width="398" alt="Screen Shot 2022-03-28 at 5 29 22 PM" src="https://user-images.githubusercontent.com/35148959/160497791-b685a429-fabb-48a6-8152-05d353e430f9.png">

I'm wondering if we need to use the controls element in this variation. I think the controls is just to separate the toggle icon from the text, though styling a generic controls element instead of the toggle icon would allow us to put other things in there besides the toggle icon and keep the spacing tied to a generic container. Doesn't seem like that applies in this variation where the only thing in the button is a single icon, since it doesn't need to be spaced apart from anything.

@mattnolting wdyt?